wallet/src/de/schildbach/wallet/di/DataSyncStatusModule.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.schildbach.wallet.di
+
+import android.content.Context
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import de.schildbach.wallet.ExploreDataSyncProgress
+import org.dash.wallet.features.exploredash.repository.DataSyncStatus
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object DataSyncStatusModule {
```

Comment:
Admittedly I have no idea myself how to better organize Hilt modules, but I think for this small service it might be unnecessary to create a new file. `DataProviderModule` might be suitable enough to have `DataSyncStatus` in it.

Reply:
I will move it to `DataProviderModule`